### PR TITLE
DOC-525 | Improve headlines and lead paragraphs in ArangoGraph

### PR DESCRIPTION
**Pull request by nerpaula**

### CODE REVIEW

File: arangograph/api-getting-started.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: default
-description: Quick start guide on how to set up a connection to the ArangoGraph API.
-title: Getting Started with the ArangoGraph Insights Platform API
+description: >-
+   Quick start guide on how to set up a connection to the ArangoGraph API
 ---
-# Getting Started with the ArangoGraph Insights Platform API
+# Getting Started with ArangoGraph API
```
Comment (Simran-B) [inline arangograph/api-getting-started.md:6]: the article should stay

File: arangograph/backups.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 ---
 layout: default
-description: Quick guides about backups in the ArangoGraph Insights Platform
-title: Backups in the ArangoGraph Insights Platform
+description: >-
+   You can manually create backups or use a backup policy to schedule periodic
+   backups and both ways allow you to store your backups in multiple regions simultaneously
```
Comment (Simran-B) [inline arangograph/backups.md:5]:
```suggestion
   backups, and both ways allow you to store your backups in multiple regions simultaneously
```

File: arangograph/notebooks.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 description: >-
-  Colocated Jupyter Notebooks within the ArangoGraph Insights Platform
+  Colocated Jupyter Notebooks within ArangoGraph, how to create and manage them
```
Comment (Simran-B) [inline arangograph/notebooks.md:4]: Maybe like this?
```suggestion
  How to create and manage colocated Jupyter Notebooks within ArangoGraph
```

File: arangograph/organizations.md
Comment (Simran-B) [inline arangograph/organizations.md:1]: Does the dashboard still call it "tier", even though this is now called "packages"?

Comment (nerpaula) [reply arangograph/organizations.md:1]: The Pricing page shows the "packages", but the Organization page still calls them "tiers".

**Review by Simran-B — APPROVED** [review]

**Pull request merged**